src/elements/common/sub-header/__tests__/Sort.test.js

```diff
@@ -142,5 +142,6 @@ describe('elements/common/sub-header/Sort', () => {
                 .childAt(1)
                 .prop('id'),
         ).toBe(messages.dateDESC.id);
+        expect(options.at(3).prop('isSelected')).toBe(true);
```

Comment:
Should you check the other options are not selected?

Reply:
I can make sure that one of them isn't selected, but checking all of them seems a bit redundant